tracer/test/Datadog.Trace.Tests/CorrelationIdentifierTests.cs

```diff
@@ -59,7 +59,14 @@ public void ServiceIdentifiers_MatchTracerInstanceSettings()
                 Environment = env
             };
             var tracer = new Tracer(settings);
-            Tracer.Instance = tracer;
+            if (Ci.CIVisibility.IsRunning)
+            {
+                Tracer.UnsafeSetTracerInstance(tracer);
```

Comment:
Do we need the condition? We could use `Tracer.UnsafeSetTracerInstance(tracer);` in either case right?

Reply:
Nope we don't need it, done in b517496